src/less/drawer-dialog/drawer-dialog.less

```diff
@@ -66,11 +66,13 @@
 
 // inherits from .icon-btn
 // Added icon button selector so it doesn't override fake-link
-button.icon-button.drawer-dialog__close {
-    background-color: transparent;
+button.icon-btn.drawer-dialog__close {
+    align-self: start;
```

Comment:
Why is the `background-color: transparent` being removed? Seems out of scope.

Reply:
Same as above, this selector was previously doing nothing since `icon-button` should actually be `icon-btn` so I pulled in current `lightbox-dialog` styles.